test/functional/helpers.lua

```diff
@@ -271,7 +271,12 @@ local function expect(contents)
   return eq(dedent(contents), curbuf_contents())
 end
 
-clear()
+function once(fn)
```

Comment:
Wouldn't an `assert(fn)` make sense here? Cause when you accidentally pass `nil` (which happens quite easy in Lua) the `if` below will lead to silently doing nothing.